nixos/modules/system/boot/kernel.nix

```diff
@@ -67,6 +68,19 @@ in
       description = "A list of additional patches to apply to the kernel.";
     };
 
+    boot.kernel.randstructSeed = mkOption {
+      type = types.str;
```

Comment:
It'd be neat to be able to read the seed from a file instead (e.g., if I want to share my config but not the seed)